src/compilerWrapper.ts

```diff
@@ -368,4 +370,47 @@ function getABIDeclaration(astRoot): { contract: string, abi: Array<ABIEntity> }
 		contract: mainContract['name'],
 		abi: interfaces
 	};
-}
\ No newline at end of file
+}
+
+export function getPlatformsCryptc() : string {
+	switch (os.platform()) {
+		case "win32":
+			return "compiler/scryptc/win32/scryptc.exe";
+		case "linux":
+			return "compiler/scryptc/linux/scryptc";
+		case "darwin":
+			return "compiler/scryptc/mac/scryptc";
+		default:
+			throw "sCrypt don't support your OS now";
+	}
+}
+
+
+
+export function getDefaultsCryptc(): string {
+	const homedir = os.homedir();
+	const extensions =  join(homedir, ".vscode/extensions");
+	if(!existsSync(extensions)) {
+		throw `No vscode extensions found, Please check if vscode is installed on your machine.`
```

Comment:
throw `No Visual Studio Code extensions found. Please ensure Visual Studio Code is installed.`